README.md

```diff
@@ -105,4 +105,11 @@ We [share our roadmap openly](./ROADMAP.md) so anyone in the community can provi
 
 ---
 
-<a href="https://trackgit.com">Trackgit</a>
+<div align="center" style="align-items: center;">
+        <a href="#top">
+            <img src="https://img.shields.io/badge/Back%20to%20Top-000000?style=for-the-badge&logo=github&logoColor=white" alt="Back to Top">
```

Comment:
A cleaner link here would be
```
https://img.shields.io/badge/Back_to_Top-black?style=for-the-badge&logo=github&logoColor=white
```

(using `_` instead of space characters, and `black` instead of the hex code `000000`

Reply:
changes made sir as you asked